packages/grafana-ui/src/utils/table.ts

```diff
@@ -28,6 +28,7 @@ export const getCellLinks = (field: Field, row: Row) => {
           origOnClick!(event, {
             field,
             rowIndex: row.index,
+            frame,
           });
```

Comment:
According to the type (https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/dataLink.ts#L96) change you made this becomes incorrect, isn't it? `origin` is typed as `T` instead of `any`.

Also, looking at this PR I think you are missing the core required change which is `DataLinkClickEvent`. Think we need to update that type so that click handlers origin is typed coreectyl as `{ frame, field, rowIndex }`. This way it should be easy to apply these changes not only to table panel, but all visualizations.

Reply:
ah, ignore the first paragraph in the comment above.